webui/react/src/pages/Cluster.module.scss

```diff
@@ -0,0 +1,10 @@
+.pivoter {
+  [role='tablist'] > div {
+    &:global(.ant-tabs-nav-wrap)::before {
+      box-shadow: none;
+    }
+    > div {
+      transform: translate(0) !important;
```

Comment:
is that possible not to use `!important`?

Reply:
unfortunately this has to override an element-level style from antd.Pivot
copy from https://github.com/determined-ai/determined/pull/7941